packages/twenty-front/src/modules/action-menu/actions/global-actions/workflow-run-actions/components/WorkflowRunActionEffect.tsx

```diff
@@ -30,7 +30,7 @@ export const WorkflowRunActionEffect = () => {
       addActionMenuEntry({
         type: 'workflow-run',
         key: `workflow-run-${activeWorkflowVersion.id}`,
-        label: capitalize(activeWorkflowVersion.workflow.name),
+        label: capitalize(activeWorkflowVersion?.workflow?.name),
```

Comment:
If the string provided to capitalize can be undefined, wouldn't it be better not to inject the action? I'm not sure how an action with an empty label is displayed.

Reply:
According to TypeScript, these properties can not be undefined or null. It is really possible for them to be undefined? If so, it might be great to update the type accordingly.

Reply:
I had some issues locally, that's why I did that. But you're right, let's not add code that should not be reached